website/views.py

```diff
@@ -377,6 +377,11 @@ def redirect_to_cos_news(**kwargs):
     return redirect('https://cos.io/news/')
 
 
+def redirect_to_registration_workflow(**kwargs):
+    # Redirect to prereg help guide
+    return redirect('https://www.cos.io/initiatives/prereg')
```

Comment:
should be  https://osf.io/registries/osf/new

Reply:
this still needs to be returned to osf.io/registries/new